config/application.rb

```diff
@@ -121,8 +121,11 @@ class Application < Rails::Application
     config.samson.ldap.bind_dn = ENV["LDAP_BINDDN"].presence
     config.samson.ldap.password = ENV["LDAP_PASSWORD"].presence
 
+    # Configuration for Gitlab
     config.samson.gitlab = ActiveSupport::OrderedOptions.new
     config.samson.gitlab.web_url = deprecated_url.call("GITLAB_URL") || 'https://gitlab.com'
+    config.samson.gitlab.api_url = deprecated_url.call("GITLAB_API_URL") || 'https://gitlab.com/api/v4'
```

Comment:
don't need deprecated_url for new urls

Reply:
the api url can just come from the web_url + api/v4 ? (inside the code ... no need to configure it afaik)